src/main/java/hudson/plugins/audit_trail/AuditLogger.java

```diff
@@ -18,4 +18,8 @@ public Descriptor getDescriptor() {
         return Jenkins.getInstance().getDescriptorOrDie(getClass());
     }
 
+    public void cleanUp() {
```

Comment:
I would suggest to allow propagating exceptions in this new method. Some Javadoc would be also great

Reply:
@oleg-nenashev thanks! Added some javadoc, as for the exception to propagate, I added the `SecurityException` that might indeed be thrown. As this is a runtime exception it's purely documentation, but at least it's aligned with the rest of the API.